tests/ordered-elements_test.js

```diff
@@ -2,95 +2,95 @@ import {assert} from 'chai';
 
 import OrderedElements from '../src/ordered-elements';
 
-describe("OrderedElements", () => {
-    it("can identify elements it has", () => {
+describe('OrderedElements', () => {
```

Comment:
not sure why all these quotes changed

Reply:
Just making things consistent. I can change this back if you like.